### PR TITLE
App skeleton

### PR DESCRIPTION
**Pull request by mglowinski93**
Pull Request contains:
- basic draw of folders structure
- basic view (127.0.0.1:5000/)
- basic api response (http://127.0.0.1:5000/api/v1.0/hello-world/)

Whole concept is based on [blueprints](https://flask.palletsprojects.com/en/1.1.x/blueprints/).
More can be found under this [book](https://coddyschool.com/upload/Flask_Web_Development_Developing.pdf#%5B%7B%22num%22%3A668%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2Cnull%2C588.58502%2Cnull%5D)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#pycharm
+.idea/
```
Comment (orzeech) [inline .gitignore:132]: you might want to add more to .gitignore using this https://www.toptal.com/developers/gitignore

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# gh-rober
\ No newline at end of file
+# Github Robber
+
```
Comment (orzeech) [inline README.md:2]: add 1-2 sentences about what the app is doing

File: README.md [hunk 3]
```diff
@@ -1 +1,6 @@
...
+###How to start application
+    1) Navigate to home folder
+    2) pip install -r requirements.txt
```
Comment (orzeech) [inline README.md:5]: use markdown syntax ` ` to indicate that this is some code

**Review by orzeech — CHANGES_REQUESTED** [review]

**Comment by orzeech** [comment]
Good job!!

File: .gitignore (hunk 1 above)
Comment (mglowinski93) [reply .gitignore:132]: According to https://www.toptal.com/developers/gitignore/api/pycharm it's _.idea_ folder, which should be added to _.gitignore_ for PyCharm. All the recommended others folders/files were already added to _.gitignore_.

**Review by orzeech — APPROVED** [review]

**Review by orzeech — CHANGES_REQUESTED** [review]
Move the backend to its own folder (leave only gitignore and readme in the main folder)

**Comment by mglowinski93** [comment]
> Move the backend to its own folder (leave only gitignore and readme in the main folder)

**Comment by mglowinski93** [comment]
> Move the backend to its own folder (leave only gitignore and readme in the main folder)

I've done this in [this commit](https://github.com/Clanhouse/gh-robber/pull/2/commits/4d665513c788d0df517dca64292b4009147f8212), but way of starting the app has changed. I'm not sure if it's fine for Windows users.

**Review by orzeech — APPROVED** [review]

**Pull request merged**